src/webportal/src/app/job/job-view/fabric/JobList/Table.jsx

```diff
@@ -0,0 +1,185 @@
+        setOrdering(new Ordering());
+      } else {
+        setOrdering(new Ordering(field, true));
+      }
+    } else {
+      setOrdering(new Ordering(column.key));
+    }
+  }
+
+  /**
+   * @param {import('office-ui-fabric-react').IColumn} column
+   */
+  function applySortProps(column) {
+    column.isSorted = ordering.field === column.key;
+    column.isSortedDescending = ordering.descending;
+    column.onColumnClick = onColumnClick;
+    return column;
+  }
+
+  const nameColumn = applySortProps({
+    key: 'name',
+    minWidth: 0,
+    fieldName: 'name',
+    isRowHeader: true,
+    isFiltered: filter.keyword !== '',
+    onRender(job) {
+      const {legacy, name, namespace, username} = job;
+      const href = legacy
+        ? `/job-detail.html?jobName=${name}`
+        : `/job-detail.html?username=${namespace || username}&jobName=${name}`;
+      return <Link href={href}>{name}</Link>;
+    },
+  });
+  const modifiedColumn = applySortProps({
+    key: 'modified',
+    minWidth: 200,
+    name: 'Date Modified',
+    isSorted: ordering.field === 'modified',
+    isSortedDescending: !ordering.descending,
+    onRender(job) {
+      return DateTime.fromJSDate(getModified(job)).toLocaleString(DateTime.DATETIME_SHORT_WITH_SECONDS);
+    },
+  });
+  const userColumn = applySortProps({
+    key: 'user',
+    minWidth: 0,
+    name: 'User',
+    fieldName: 'username',
+    isFiltered: filter.users.size > 0,
+  });
+  const durationColumn = applySortProps({
+    key: 'duration',
+    minWidth: 0,
+    name: 'Duration',
+    onRender(job) {
+      return Duration.fromMillis(getDuration(job)).toFormat(`h:mm:ss`);
+    },
+  });
+  const virtualClusterColumn = applySortProps({
+    key: 'virtualCluster',
+    minWidth: 0,
+    name: 'Virtual Cluster',
+    fieldName: 'virtualCluster',
+    isFiltered: filter.virtualClusters.size > 0,
+  });
+  const statusColumn = applySortProps({
+    key: 'status',
+    minWidth: 0,
+    name: 'Status',
+    isFiltered: filter.statuses.size > 0,
+    onRender(job) {
+      /** @type {React.CSSProperties} */
+      const wrapperStyle = {display: 'inline-block', verticalAlign: 'middle'};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const statusText = getStatusText(job);
+      /** @type {MessageBarType} */
+      const messageBarType = {
+        Waiting: MessageBarType.warning,
+        Running: MessageBarType.info,
+        Stopping: MessageBarType.severeWarning,
+        Succeeded: MessageBarType.success,
+        Failed: MessageBarType.error,
+        Stopped: MessageBarType.blocked,
+      }[statusText];
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconContainerStyle = {marginTop: 8, marginBottom: 8, marginLeft: 8};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const textStyle = {marginTop: 8, marginRight: 8, marginBottom: 8};
+      return (
+        <div style={Object.assign(wrapperStyle, zeroPaddingRowFieldStyle)}>
```

Comment:
You can use column's classname to override cell's padding.
https://github.com/Microsoft/pai/blob/d13705706efe6659ab3b8fb6afc47232cc15198c/src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx#L281